src/lang/po/fr/Prusa-Firmware-Buddy_fr.po

```diff
@@ -326,7 +326,7 @@ msgstr ""
 "Version du Bootloader\n"
 "%d.%d.%d\n"
 "\n"
-"Buddy Board\n"
```

Comment:
This should be "Buddy Board" as it refers to the xBuddy board, thus shouldn't be translated.

Reply:
fixed

Reply:
This does not appear fixed